extensions/physics-nodes/CCPhysicsDebugNodeBox2D.h

```diff
@@ -35,6 +35,66 @@ struct b2AABB;
 
 extern cocos2d::DrawNode* drawBox2D;
 
+/// Test settings. Some can be controlled in the GUI.
+struct Settings
```

Comment:
Name style suggetion: Settings to Box2dDrawSettings and put to namespace cocos2d::extensions maybe better